README.md

```diff
@@ -18,40 +18,23 @@ body) of requests. It also does not aim to be the fastest, or most secure.
 
 Java (version 16 or later). No third-party libraries required for execution.
 
-Dependencies for build / test (such as JUnit) are managed using Gradle. All dependencies should be current, stable
-versions.
+## Running
 
-## Building
```

Comment:
So only contributors will want to build it?

Reply:
I don't see why an end user would want to build it. Unless they wanted to customize the behavior.

Reply:
Apparently, there are some people who like running applications before they are released.

Reply:
Host pre-releases then. If I wanted to know how to build/use from a main branch I would look in contributing.

But w/e it's not really a big deal.